core/imageroot/var/lib/nethserver/cluster/actions/change-user-password/validate-input.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "change-user-password-input",
+  "$id": "http://schema.nethserver.org/cluster/change-user-password-input.json",
+  "description": "Input schema of the change-user-password action",
+  "examples": [
+    {
+      "user": "admin",
+      "current_password": "Nethesis,1234",
+      "new_password": "Nethesis,12345"
+    }
+  ],
+  "type": "object",
+  "properties": {
+    "user": {
+      "description": "Target user.",
+      "type": "string"
+    },
+    "current_password": {
+      "description": "Current password in plain text",
+      "type": "string"
+    },
+    "new_password": {
+      "description": "New password in plain text. The password will be updated only the authentication with 'user' and 'current_password' si valid.",
+      "type": "string"
+    }
+  },
+  "required": [
+    "user",
+    "current_password",
+    "new_password"
```

Comment:
```suggestion
    "new_password",
  ]
```

Reply:
The comma is not required.

Reply:
Right, the square bracket is required, though!